apis/r/copy_source.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-test -d ../../libtiledbsoma && cp -a ../../libtiledbsoma/ src/
+test -d ../../libtiledbsoma && cp -a ../../libtiledbsoma src/
```

Comment:
The darn `cp` command on macOS had tripped me up before.  It complains about the trailing slash?

Reply:
Ugh yeah, what a pain. I documented exactly what was going b/c I've also been bit by this before.